docs/castor/linear-problem/ra-usage-limits-filler.md

```diff
@@ -2,36 +2,43 @@
-| TSOs                           | $tso \in \mathcal{TSO}$               | Set of all TSOs operating a range action in RangeActions                                                                                                                                                                                                 |
+| Name                                         | Symbol                                | Details                                                                                                                                                                                                                                                  |
+|----------------------------------------------|---------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| RangeActions                                 | $r,s \in \mathcal{RA}$                | Set of optimised RangeActions and state on which they are applied (PSTs, HVDCs, InjectionRangeActions...)                                                                                                                                                |
+| PstRangeActions                              | $r \in \mathcal{RA}^{PST}$            | Set of PST RangeActions                                                                                                                                                                                                                                  |
+| States                                       | $s \in \mathcal{S}$                   | Set of states on which RangeActions are optimized                                                                                                                                                                                                        |
+| PrePerimeterSetpoints                        | $\alpha _0(r)$                        | Setpoint of RangeAction $r$ at the beginning of the optimisation                                                                                                                                                                                         |
+| Range upper bound                            | $\underline{\alpha(r)(\alpha _0(r))}$ | Highest allowed setpoint for a range action $r$, given its pre-perimeter setpoint $\alpha _0(r)$                                                                                                                                                         |
```

Comment:
> does underline and overline work here?

(at least on my computer) it no longer works since we migrated the documentation, that's why I replaced it

Reply:
I used `alpha_{\max}(r)(\alpha _0(r)` and `\alpha_{\min}(r)(\alpha _0(r))` instead